### ISSUE TITLE
A new libjingle release

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yss163com/libjingle#212]
```
Purpose of code changes on this branch:
Release the new changes needed mostly by PeerConnection:
  - Add dummydevicemanager.
  - Remove PeerConnection OnLocalStreamInitialized callback.
  - Fix webrtc_json.cc numeric locale formatting issue.
  - Don't start playout until the local content has been set.

When reviewing my code changes, please focus on:


After the review, I'll merge this branch into:
/trunk


```

Original issue reported on code.google.com by `ronghuawu@google.com` on 14 Sep 2011 at 9:44
- Merged into: #213

Attachments:
- [libjingle_release.diff](https://storage.googleapis.com/google-code-attachments/libjingle/issue-212/comment-0/libjingle_release.diff)
